src/reactviews/pages/QueryResult/queryResultPane.tsx

```diff
@@ -100,6 +100,15 @@ const useStyles = makeStyles({
     },
 });
 
+const MIN_GRID_HEIGHT = 273; // Minimum height for a grid
```

Comment:
Is this a random pick or did we use the same number in ADS?

Reply:
Using the same number as ADS